engine/class_modules/warlock/sc_warlock_affliction.cpp

```diff
@@ -935,7 +934,9 @@ void warlock_t::create_buffs_affliction()
 
   buffs.malefic_wrath = make_buff( this, "malefic_wrath", find_spell( 337125 ) )->set_default_value_from_effect( 1 );
 
+  // TODO: Replace hardcoded duration with query from spell data, replace buff spell (363953) with actuall buff spell ID
   buffs.calamitous_crescendo = make_buff( this, "calamitous_crescendo", find_spell( 363953 ) )
+                                   ->set_duration(timespan_t::from_seconds(10))
```

Comment:
If you want, you can use `10_s` instead of `timespan_t::from_seconds( 10 )`

Reply:
Appreciate the advice, in the future I will try to put it to use but I do not intend to set these values.